x-pack/plugins/secops/public/components/page/network/kpi_network/index.tsx

```diff
@@ -22,34 +23,44 @@ import * as i18n from './translations';
 
 interface KpiNetworkProps {
   data: KpiNetworkData;
+  loading: boolean;
 }
 
-const kpiNetworkCards = (data: KpiNetworkData) => [
+const loadMatrix = (loading: boolean, data: KpiNetworkData, property: string) => {
+  const matrix: number | null | undefined = get(property, data);
+
+  if (typeof matrix !== 'undefined' && matrix !== null) {
```

Comment:
You can use `if (matrix != null)` if you are wanting to ensure matrix is neither a `null` or `undefined` and should do what you want.

Reply:
yeah, good point. It looks nicer in that way. I was struggling with which role I should follow. Some people don't like loose equal in the code but if it is how we handle it, should be fine.